server.py

```diff
@@ -550,6 +554,41 @@ def get(self):
 
         return return_dict
 
+class PendingMergesResource(restful.Resource):
+    def get(self):
+        #TODO: decide if we want to implement this
+        pass
+
+    def post(self, region):
+        dao = Dao(region, mongo_client=mongo_client)
+        requesting_user = get_user_from_access_token(request.headers, dao)
+        if not is_user_admin_for_region(requesting_user, region): #wow, such auth
```

Comment:
sorry i forgot to reply to your last comment, but can you change this to `if not request_user.admin_regions`?